### ISSUE TITLE
Package name conflict with pyload.utils on PyPI

### ISSUE THREAD

**Issue by vuolter** [thammi/pyload-utils#2]
Hello, since pyLoad finally came on pypi we had to face the possibility to encounter issues of naming conflict where the best suitable name for the package has been already taken, like this case.
Now pyLoad is released as a namespace package, that's mean we are many modules (with many names), but, luckily, we was able to get really plain names for almost all the modules we are porting to pypi, like `pyload.core` for the main program, `pyload.webui` for the web interface and so on. Actually, just the utils package could not be registered as `pyload.utils` obviously because of your package's name.
In short, I'd be very grateful to you if you could rename this package's name on pypi to something else and transfer the owership of its current name to me (the pyload mantainer btw), you'd definitely simplify the our process of code releasing and make a little less harder for the users to identify the main pyload package.
Thank you!
